cluster/saltbase/salt/kube-addons/kube-addon-update.sh

```diff
@@ -471,28 +406,6 @@ function reconcile-objects() {
     fi
 }
 
-function update-addons() {
```

Comment:
I think you intended to delete 'update-objects' and 'update-object' not this method.